degiro_connector/quotecast/models/quotecast_parser.py

```diff
@@ -267,12 +267,25 @@ def build_ticker_from_quotecast(
             if message["m"] == "un":
                 reference = message["v"][0]
                 value = message["v"][1]
-                product, metric = references[reference]
-                ticker.products[product].metrics[metric] = value
+                pm = references.get(reference)
+                if pm is not None:
```

Comment:
I think we can use this simplication here:
```python
if reference in references:
    ...
```

Reply:
Thanks for your suggestion, simplifications implemented.